src/components/Typography/Typography.tsx

```diff
@@ -1,19 +1,28 @@
-import './Typography.scss';
-
 import classNames from 'classnames';
 import { ReactNode } from 'react';
+
+// FontVariant for the new categories and variants
 type FontVariant =
-  | 'tiny'
-  | 'small'
-  | 'small-medium'
-  | 'small-medium-400'
-  | 'medium'
-  | 'medium-large'
-  | 'large'
-  | 'extra-large'
-  | 'extra-large-600'
-  | 'extra-extra-large-500'
-  | 'extra-extra-large';
+  | 'D1'
```

Comment:
nit: these should be lower case:
```suggestion
  | 'd1'
```

Reply:
^